capstone/src/test/java/com/google/sps/data/DataStoreTest.java

```diff
@@ -0,0 +1,46 @@
+import static com.google.appengine.api.datastore.FetchOptions.Builder.withLimit;
+import static org.junit.Assert.assertEquals;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import org.junit.After;
+import org.junit.Before;
+
+public class DataStoreTest {
+
+  private final LocalServiceTestHelper helper =
+      new LocalServiceTestHelper(new LocalDatastoreServiceTestConfig());
+
+  @Before
+  public void setUp() {
+    helper.setUp();
+  }
+
+  @After
+  public void tearDown() {
+    helper.tearDown();
+  }
+
+  // Run this test twice to prove we're not leaking any state across tests.
+  private void doTest() {
+    DatastoreService ds = DatastoreServiceFactory.getDatastoreService();
+    assertEquals(0, ds.prepare(new Query("test")).countEntities(withLimit(10)));
+    ds.put(new Entity("test"));
+    ds.put(new Entity("test"));
+    assertEquals(2, ds.prepare(new Query("test")).countEntities(withLimit(10)));
+  }
+
+  //   @Test
```

Comment:
Any reason why these are commented out? Can you add a comment as to why these are commented out if this is intentional?

Reply:
Yeah I accidentally had the DataStore test class in this PR, it was preventing merging since the tests failed. I intend to add testing after demo week

Reply:
I'll add a comment describing why